templates/repo/view_file.tmpl

```diff
@@ -120,22 +120,12 @@
 						{{end}}
 					</tbody>
 				</table>
-				<div class="code-line-menu ui fluid popup transition hidden">
-					<div class="ui column relaxed equal height">
-						<div class="column">
-							{{if $.Permission.CanRead $.UnitTypeIssues}}
-								<div class="ui link list">
-									<a class="item ref-in-new-issue" href="{{.RepoLink}}/issues/new?body={{.Repository.HTMLURL}}{{printf "/src/commit/" }}{{PathEscape .CommitID}}/{{PathEscapeSegments .TreePath}}" rel="nofollow noindex">{{.locale.Tr "repo.issues.context.reference_issue"}}</a>
-								</div>
-							{{end}}
-							<div class="ui link list">
-								<a class="item view_git_blame" href="{{.Repository.HTMLURL}}/blame/commit/{{PathEscape .CommitID}}/{{PathEscapeSegments .TreePath}}">{{.locale.Tr "repo.view_git_blame"}}</a>
-							</div>
-							<div class="ui link list">
-								<a data-clipboard-text="{{.Repository.HTMLURL}}/src/commit/{{PathEscape .CommitID}}/{{PathEscapeSegments .TreePath}}" class="item copy-line-permalink">{{.locale.Tr "repo.file_copy_permalink"}}</a>
-							</div>
-						</div>
-					</div>
+				<div class="code-line-menu ui vertical pointing menu hide">
+					{{if $.Permission.CanRead $.UnitTypeIssues}}
+						<a class="item ref-in-new-issue" href="{{.RepoLink}}/issues/new?body={{.Repository.HTMLURL}}{{printf "/src/commit/" }}{{PathEscape .CommitID}}/{{PathEscapeSegments .TreePath}}" rel="nofollow noindex">{{.locale.Tr "repo.issues.context.reference_issue"}}</a>
+					{{end}}
+					<a class="item view_git_blame" href="{{.Repository.HTMLURL}}/blame/commit/{{PathEscape .CommitID}}/{{PathEscapeSegments .TreePath}}">{{.locale.Tr "repo.view_git_blame"}}</a>
+					<a class="item copy-line-permalink" data-url="{{.Repository.HTMLURL}}/src/commit/{{PathEscape .CommitID}}/{{PathEscapeSegments .TreePath}}">{{.locale.Tr "repo.file_copy_permalink"}}</a>
```

Comment:
IIRC, the `data-url` are used for AJAX requests in Gitea code base.

Here the URL is just for clipboard copying, so the old name `data-clipboard-text` is more accurate.

(Nit, not blocker from my side)

Reply:
Problem is `data-clipboard-text` is handled by other clipboard code and I wanted to avoid that interaction and do it manually with that new event handler.